data/mods/Magiclysm/enchanted/enchanted_melee.json

```diff
@@ -136,7 +136,7 @@
   },
   {
     "type": "GENERIC",
-    "id": "metal_bat_plus_two",
+    "id": "bat_metal_plus_two",
```

Comment:
This is going to break saves where the item isn't present.
If it's WIP and regular games can't have it yet anyway that's fine, but if there's an in-game way to acquire it, it's not ok and needs a migration entry.

Reply:
They never spawned, wasn't in loot lists until now. we have our own obsolete file for things like that.

Reply:
although... hey @LaVeyanFiend , you might want to check and see if those metal bats were used in your MA thing, you may remember off the top of your head. One was a typo.

Reply:
Baseball bats aren't used in any MA, so I wouldn't have copied the magical ones in.